app/search.py

```diff
@@ -0,0 +1,53 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+class YTSearch:
+    """
+    Formatting for retrieving youtube search results for a given query
+    """
+
+    def __init__(self, query):
+        self.params = {
+            "videoCategoryId": "10",
+            "part": "snippet",
+            "type": "video",
+            "q": query,
+            "videoEmbeddable": True,
+            "key": os.getenv("YOUTUBE_KEY"),
+        }
+        self.root_url = "https://www.googleapis.com/youtube/v3/search"
+        self.results = self.get_results()
+        self.matches = self.results['result']['items']
+
+    def get_results(self):
+        resp = requests.get(self.root_url, self.params)
+        youtube_results = resp.json()
+
+        return {"resp": resp, "result": youtube_results}
+
+
+class LastFMSearch:
+    """
+    Formatting for retrieving last.fm search results for a given query
+    """
+
+    def __init__(self, query):
+        self.params = {
+            "method": "track.search",
+            "format": "json",
+            "track": query,
+            "api_key": os.getenv("LASTFM_KEY"),
+        }
+        self.root_url = "http://ws.audioscrobbler.com/2.0/"
```

Comment:
This should probably live in a constants file somewhere and be injected into the constructor of this class to make it more configurable. You could have the constructor take `audio_api_url=constants.AUDIO_SCROBBLER_API_URL` as a default kwarg